Advanced-Recruit-Training/src/BatteryStateDisplayService.cpp

```diff
@@ -27,7 +27,6 @@ void BatteryStateDisplayService::handleBatteryDataReceived(const BatteryData& ba
 
     QTextStream(stdout) << "Voltage: " << batteryData.voltage
         << " Current: " << batteryData.current
-        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed() << endl;
-
-    // TODO: Print out time until the battery is fully charged or depleted.
+        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed()
+        << " Time Remaining: " << batteryStateOfChargeService_.timeWhenChargedOrDepleted().toString("hh:mm:ss.zz") <<endl;
```

Comment:
This function should tell us the time remaining _until the battery is charged or depleted_, just having the time doesn't tell us much.